### PR TITLE
link hosted https://www.jslint.com/help.html

### LINKED ISSUE #289: link rendered help.html
> help.html describes JSLint's usage. Please read it.

But https://github.com/jslint-org/jslint/blob/master/help.html is not readable in raw form.

Yes, I can clone repo - but linking https://www.jslint.com/help.html would be nice

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes #289

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ that sets options.
 
 `report.js` generates the results reports in HTML.
 
-`help.html` describes JSLint's usage. Please read it.
+`help.html` describes JSLint's usage. Please [read it](https://www.jslint.com/help.html).
```
Comment (kaizhu256) [inline README.md:35]:
```suggestion
`help.html` describes JSLint's usage. Please [read it](https://jslint-org.github.io/jslint/help.html).
```

**Comment by kaizhu256** [comment]
thx @matkoniecz ! in future, try to make pull-requests to `beta` branch ; )

**Comment by matkoniecz** [comment]
ops, sorry

Maybe put it into CONTRIBUTING.md file? See https://github.blog/2012-09-17-contributing-guidelines/

**Comment by kaizhu256** [comment]
i've decided to make branch-beta the default-branch.  that will default all pr's to it (w/o requireing ppl to read the instructions first ; )

**Pull request merged**